### DISCUSSION TITLE
Welcome to docker-qgis-server-qwc2 Discussions!

### DISCUSSION THREAD

**Discussion by bor8** [bor8/docker-qgis-server-qwc2#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sissi086** [comment]
Come faccio a caricare delle mappe su qwc2 e mandarle a scherno? Io ho scaricato delle mappe da geoserver ma nin riesco a visualizzare la mappa su questo qwc2 sapete aiutarmi
